### PR TITLE
add package pipes.sh

### PR DESCRIPTION
**Pull request by Efreak**
A pipes screensaver for console.

Screenshot_20211218-154117_Termux

### CODE REVIEW

File: packages/pipes.sh/build.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/pipeseroni/pipes.sh
+TERMUX_PKG_DESCRIPTION="Animated pipes terminal screensaver"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@Efreak"
+TERMUX_PKG_VERSION=1.3.0+ #if there's ever a new release, switch to it
```
Comment (xeffyr) [inline packages/pipes.sh/build.sh:5]: Remove "+".

If you prefer git version or package doesn't provide tags, use commit date here or date when you have added/upgraded package.

File: packages/pipes.sh/build.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+TERMUX_PKG_SRCURL=https://github.com/pipeseroni/pipes.sh/archive/refs/heads/master.zip # in the meantime, master has a bunch of changes
```
Comment (xeffyr) [inline packages/pipes.sh/build.sh:6]: Source code downloads must be deterministic. Please use tags (if possible) or specific commit. `master.zip` will change over time and will make git history for that package useless.

**Review by xeffyr — CHANGES_REQUESTED** [review]

**Comment by Efreak** [comment]
Somehow, this isn't making a .deb file still.

**Comment by buttaface** [comment]
Maybe [it has a problem with the `.` in the package name](https://github.com/termux/termux-packages/runs/4573808528?check_suite_focus=true#step:3:122)?

**Comment by xeffyr** [comment]
@buttaface Debian packages allow `.` (dot) in package name.

**Comment by buttaface** [comment]
Could be a problem on our CI first, see the error I linked.

**Comment by xeffyr** [comment]
I've pushed fix for regexp in github actions config.

**Pull request merged**